changelog/2022-03-09.md

```diff
@@ -0,0 +1,30 @@
+
+## :wrench: March 9, 2022
+
+**CLI version: 0.8.4**
+
+:heavy_check_mark: Resolves an issue where using a relative path for the `dependency_file_info` property in a Run config was breaking. For example, this now works: 
```

Comment:
i would add "this now works if you were operating from a subdirectory of a git repo" or something analogous.